x-pack/test/pki_api_integration/fixtures/es_ca.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
How did we get access to this? I assume this is just checked in so we can generate other certificates for users?

Reply:
> How did we get access to this? 

Mike planned to [check it in initially](https://github.com/elastic/kibana/pull/42527/commits/0a562e721676003bed3ac423e75fc1a78cb6d06c#diff-b0b811d7306fae7507d0201f1f225ed3), but didn't find a use case for that.

> I assume this is just checked in so we can generate other certificates for users?

Yeah, I've checked it in just to be able to sign test certificates in the future if we need them. Since this CA is used to sign pre-packaged dev certs we use to start ES with TLS in @kbn/es. We can live without it if you think it's not needed.

Reply:
I like it being checked in, it makes use-cases like our easier. I'm tempted to suggest we move it to `@kbn/dev-utils`, but if we're the only ones using it, it likely doesn't matter